tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/Datadog.Trace.ClrProfiler.Managed.Tests.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="log4net" Version="$(Log4NetVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
     <PackageReference Include="NLog" Version="$(NLogVersion)" Condition=" $(TargetFramework.StartsWith('net4'))" />
+    <PackageReference Include="RabbitMQ.Client" Version="7.1.1" />
```

Comment:
I don't think you can use this on .NET FX can you? do you need a condition here?

Reply:
The test build and runs just fine on my machine 🤷🏽‍♂️

Reply:
The nuget package supports `netstandard2.0`
https://www.nuget.org/packages/RabbitMQ.Client#supportedframeworks-body-tab